redash/cli/database.py

```diff
@@ -41,7 +41,7 @@ def load_extensions(db):
             connection.execute(f'CREATE EXTENSION IF NOT EXISTS "{extension}";')
 
 
-@manager.command()
+@manager.command(name="create_tables")
```

Comment:
Using dash instead of underscore command in Click 7.0. https://github.com/pallets/click/issues/1123